package.json

```diff
@@ -1,8 +1,15 @@
 {
   "name": "@asteasolutions/zod-to-openapi",
-  "version": "6.4.0",
+  "version": "7.0.0",
   "description": "Builds OpenAPI schemas from Zod schemas",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/mjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/mjs/index.js",
+      "require": "./dist/cjs/index.js"
+    }
+  },
   "types": "dist/index.d.ts",
```

Comment:
Shouldn't the types be moved as well?

Reply:
Yes, that looks like something new (to me) . I followed this guide as to support nodejs and typescript for older and newer versions; https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#package-json-exports-imports-and-self-referencing